tests/acceptance/blocks/thememap.js

```diff
@@ -1,23 +1,32 @@
-import { Selector, t } from 'testcafe';
+import { Selector, t, ClientFunction } from 'testcafe';
 
 /**
  * This class models user interactions with the {@link ThemeMap}.
  */
 class ThemeMap {
   constructor () {
-    this._pin = Selector('.js-answersMap button');
+    this._pin = Selector('.yxt-Pin button');
+    this._pinCluster = Selector('.yxt-PinCluster button');
     this._canvas = Selector('.mapboxgl-canvas');
     this._searchThisAreaToggle = Selector('.Answers-searchThisAreaToggleLabel');
     this._searchThisAreaButton = Selector('.js-searchThisAreaButton');
   }
 
   /**
-   * Selects a pin from the Mapbox map.
+   * Selects a pin from the map.
    */
-  async selectMapboxPin () {
+  async selectPin () {
```

Comment:
For this and `selectPinCluster` is there any way to target a specific pin or cluster? Just curious. Doesn't seem like we need that for this PR, but we may want that for future tests.

Reply:
Yes, it should be possible to select specific pins or clusters since the different clusters and pins have different aria labels that we can use for targeting.